clients/src/test/java/org/apache/kafka/common/utils/ByteUtilsTest.java

```diff
@@ -239,6 +239,66 @@ public void testDouble() throws IOException {
         assertDoubleSerde(Double.NEGATIVE_INFINITY, 0xFFF0000000000000L);
     }
 
+    private static int mathSizeOfUnsignedVarint(int value) {
+        int leadingZeros = Integer.numberOfLeadingZeros(value);
+        // return (38 - leadingZeros) / 7 + leadingZeros / 32;
+        int leadingZerosBelow38DividedBy7 = ((38 - leadingZeros) * 0b10010010010010011) >>> 19;
+        return leadingZerosBelow38DividedBy7 + (leadingZeros >>> 5);
+    }
+
+    @Test
+    public void testSizeOfUnsignedVarintMath() {
+        for (int i = 0; i < Integer.MAX_VALUE; i++) {
```

Comment:
Is this too slow for a unit test? Similar question for other tests that do a large range of values.

Reply:
The tests I have now run in ~0.6 seconds.